syntax = "proto3";

message JoinChatRoom
{
  int32 nt_type      = 1;
  int64 user_id      = 2;
  int64 chat_room_id = 3;
}
